docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,3414 @@
+* Show alert indicator on alerts tab {kibana-pull}148048[#148048]
+* Adds latency alert history chart on the Alert details page for APM {kibana-pull}148011[#148011]
+* Adds alert annotation and threshold shade on the APM latency chart on the Alert Details page {kibana-pull}147848[#147848]
+* Errors group sampler {kibana-pull}147571[#147571]
+* Show alert indicator on service inventory page {kibana-pull}147511[#147511]
+* Adds alertDetailAppSection to the APM Rule Details page {kibana-pull}143298[#143298]
+
+Dashboard::
+* Adds the ability to load more options list suggestions when you scroll {kibana-pull}148331[#148331]
+* Adds alert filters to the Detection page {kibana-pull}146989[#146989]
+* Adds the image embeddable {kibana-pull}146421[#146421]
+* Adds the "Convert to lens" action to Dashboard {kibana-pull}146363[#146363]
+* Adds a step size to the time slider control {kibana-pull}145033[#145033]
+* Adds the ability to sort the options list suggestions {kibana-pull}144867[#144867]
+
+Elastic Security::
+For the Elastic Security 8.7.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+Enterprise Search::
+For the Elastic Enterprise Search 8.7.0 release information, refer to {enterprise-search-ref}/changelog.html[_Elastic Enterprise Search Documentation Release notes_].
+
+Fleet::
+* Adds ability to show FQDN of agents {kibana-pull}150239[#150239]
+* Adds `getStatusSummary` query parameter to `GET /api/fleet/agents` API {kibana-pull}149963[#149963]
+* Enabling diagnostics feature flag and changed query for files to use upload_id {kibana-pull}149575[#149575]
+* Experimental toggles for doc-value-only {kibana-pull}149131[#149131]
+* We now display agent metrics, CPU and memory in the agent list table and agent details page {kibana-pull}149119[#149119]
+* Implement subcategories in integrations UI {kibana-pull}148894[#148894]
+* Added rollout period to upgrade action {kibana-pull}148240[#148240]
+* Adds per-policy inactivity timeout + use runtime fields for agent status {kibana-pull}147552[#147552]
+* Show dataset combo box for input packages {kibana-pull}147015[#147015]
+* Adds UI controls to setting/outputs to configure new shipper {kibana-pull}145755[#145755]
+
+Infrastructure::
+* Adds link to ingest pipeline dashboard from Stack Monitoring {kibana-pull}149721[#149721]
+
+Integrations::
+* User friendly UX added alongside advanced yaml editor {kibana-pull}147900[#147900]
+* Custom fleet policy UX for new integration (cloud defend v1) {kibana-pull}147300[#147300]
+
+Kibana Home & Add Data::
+Self-managed {kib} instances now have a link to instructions for migrating self-managed clusters to Elastic Cloud {kibana-pull}145523[#145523]
+
+Lens & Visualizations::
+* Adds the share link feature in *Lens* {kibana-pull}148829[#148829]
+* Adds the ability to support complex filters with AND/OR relationships {kibana-pull}143928[#143928]
```

Comment:
This is available also on dashboards, discover and wherever the unified search component lives. Does it make sense to add it on a more generic section?